src/content/docs/cypher/query-clauses/call.md

```diff
@@ -27,20 +27,22 @@ The following tables lists the built-in schema functions you can use with the `C
 | `property id` | Internal identifier of the property within table | INT64 |
 | `name` | name of the property | STRING |
 | `type` | data type of the property | STRING |
+| `default expression` | default expression of the property | STRING |
 | `primary key` | if property is primary key | BOOLEAN |
 
 ```cypher
 CALL TABLE_INFO('User') RETURN *;
 ```
 Output:
 ```
----------------------------------------------
-| property id | name | type   | primary key |
----------------------------------------------
-| 0           | name | STRING | True        |
----------------------------------------------
-| 1           | age  | INT64  | False       |
----------------------------------------------
+-----------------------------------------------------------------
+| property id | name | type   | default expression |primary key |
+| INT64       |STRING| STRING | STRING             |BOOL        |
+-----------------------------------------------------------------
+| 0           | name | STRING | NULL               |True        |
```

Comment:
The meaning of `default expression` is vague if we just show NULL values - to make it more clear, can we show an example where the default expressions isn't a NULL?